### PR TITLE
Verifying installation doc update

### LINKED ISSUE #347: Updating the DART Documentation for the "Verifying Installation" page
There are some minor changes that can be made to improve the clarity and simplicity of the process on the 'Verifying Installation" page (https://docs.dart.ucar.edu/en/latest/guide/verifying-installation.html#runningSomething). Some changes to the formatting of this page as well as adding some missing information will accomplish this. Below are the details of some of the modifications to be made:

- Matlab must be called with the full path unless the user's profile has been edited (EX `/Applications/MATLAB_R2022a.app/bin/matlab -nodesktop` must be run for Mac users instead of `matlab -nodesktop`. We will also include how to determine the location in which MATLAB was installed, if it is not the default.

- It is not entirely clear in the MATLAB script which commands are to be executed by the user versus what is supposed to be outputted. Which commands the user will execute can be more explicitly stated or highlighted. We can also be more descriptive by describing what each command is doing / which model it is generating, as is done in the DART tutorial.

- There is a redundant link [Checking the build — running something](https://docs.dart.ucar.edu/en/latest/guide/verifying-installation.html#runningSomething) that leads to the current webpage.

- plot_total_err does not ask the user to 'Input name of true model trajectory file; (cr) for perfect_output.nc' as is described in the script and only prompts the user once to ‘Input name of ensemble trajectory file; (cr) for preassim.nc’.

- It is not explicitly stated on this page that the user can/should select the default file by hitting carriage return for all Matlab exercises, as it is in the DART Tutorial.

### PR DESCRIPTION
**Pull request by mjs2369**
## Description:

Changes have been made to increase clarity and usability of the Verifying Installation documentation page (https://docs.dart.ucar.edu/en/latest/guide/verifying-installation.html#runningSomething), as well as adding some missing information and fixing incorrect outputs. 

The biggest change was to the organization of the code blocks: breaking the single code block with all executable commands and subsequent output into multiple, allowing for more specific information about each command for execution in between. 

Some of the minor edits to the document are as follows:

- Matlab must be called with the full path unless the user's profile has been edited (EX /Applications/MATLAB_R2022a.app/bin/matlab -nodesktop must be run for Mac users instead of matlab -nodesktop. We will also include how to determine the location in which MATLAB was installed, if it is not the default.
- Redundant link [Checking the build — running something](https://docs.dart.ucar.edu/en/latest/guide/verifying-installation.html#runningSomething) has been removed.
- The example output of plot_total_err has been fixed to only prompt the user to ‘Input name of ensemble trajectory file; (cr) for preassim.nc’, as this corresponds to the true I/O prompts for this executable. 
- Explicitly stating on this page that the user can/should select the default file by hitting carriage return for all Matlab exercises, as it is in the DART Tutorial.
- Making sure the user has run all the required executables/scripts in order to build all of the necessary input files.

### Fixes issue
Fixes #347 

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

### Documentation changes needed?

- [ ] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly.

### Tests
Built the documentation page via Readthedocs

## Checklist for merging

- [x] Updated changelog entry
- [x] Documentation updated
- [x] Version tag 

## Testing Datasets

- [ ] Dataset needed for testing available upon request
- [ ] Dataset download instructions included
- [x] No dataset needed

### CODE REVIEW

File: guide/verifying-installation.rst [hunk 1]
```diff
@@ -28,65 +27,127 @@ time.
...
-                   truth_time: [1 200]
-                   diagn_time: [1 200]
-                         vars: {'state'}
-                         time: [200x1 double]
-           time_series_length: 200
-                          var: 'state'
-                     var_inds: [1 2 3]       
+   $ csh quickbuild.csh
+   $ ./filter
+   $ ./perfect_model_obs
```
Comment (hkershaw-brown) [inline guide/verifying-installation.rst:51]: perfect_model_obs creates observations to be used by filter, so although this order will work as is, it is
better for people's understanding of DART to switch the order of these commands

```suggestion
   $ ./perfect_model_obs
   $ ./filter
```

File: guide/verifying-installation.rst [hunk 2]
```diff
@@ -28,65 +27,127 @@ time.
...
+   /Applications/MATLAB_R20XXx.app
+
+``R20XXx`` must also be replaced with the version of MATLAB that you have
+installed. For example, the full command for a Mac user having installed the
+latest version of MATLAB would be ``/Applications/MATLAB_R2022a.app/bin/matlab -nodesktop``.
+
+If the corresponding MATLABROOT listed above does not work, you can identify the
+correct directory by opening the MATLAB application and executing the command
+``matlabroot``, which will display the correct location with the correct version
+incorporated into the MATLABROOT.
```
Comment (hkershaw-brown) [inline guide/verifying-installation.rst:80]:
```suggestion
``matlabroot``, which will display the directory where MATLAB is installed.
```

**Review by hkershaw-brown — CHANGES_REQUESTED** [review]
Looks great. I added a couple of suggestions. Only one is something to definitely change:
 ./perfect_model_obs before ./filter. 
 
The other suggestion is just a personal preference to have a less wordy sentence.

**Comment by kdraeder** [comment]
Overall it looks straightforward and helpful.
Here are some mostly minor things that caught my eye, which you can consider changing.
I haven't put them in the probably more useful format that Helen's suggestions are in. If you point me to a button or instructions about how to do that, I'll reformat my comment.

Can we suggest any alternatives to using MATLAB?

In the page rendered by github the historical links show up as
":doc:\`The Lorenz 63 model: what is it and why should we care? <lorenz-63-model>\`"
Is that expected?  Does the formatting disappear in the actual 
documentation pages?

Line 18: 
"observations sequences" -> 
"observation sequences"

Line 29 and following: 
Do we have a convention about whether the DART part of pathnames is literal or represents the user's path where DART is?
Maybe I should be asking "Did previous instructions tell users do download DART into a directory called 'DART' in their current directory?"

Line 49: 
$ csh quickbuild.csh ?->
$ csh quickbuild.sh       or are we waiting until this change has been made?

Line 96:
make the DART/matlab/ functions ?->
make the DART MATLAB functions  or
make the DART/diagnostics/matlab/ functions

Line 109:
At some level we're supposed to write in the least "triggering" way
(which itself is an example of the language we're supposed to avoid),
so "simply hit enter" could be replaced by "simply press enter"

Line 176:
 and/or myself ->  or myself

**Comment by hkershaw-brown** [comment]
@kdraeder quick answers to these two  

> Can we suggest any alternatives to using MATLAB?

Not at this time.

> In the page rendered by github the historical links show up as
> ":doc:`The Lorenz 63 model: what is it and why should we care? `"
> Is that expected? Does the formatting disappear in the actual
> documentation pages?

the built docs show up here 
https://dart-documentation--350.org.readthedocs.build/en/350/guide/verifying-installation.html

**Comment by mjs2369** [comment]
Thanks for the feedback @kdraeder @hkershaw-brown ! I will make the necessary changes. 

> Line 29 and following: Do we have a convention about whether the DART part of pathnames is literal or represents the user's path where DART is? Maybe I should be asking "Did previous instructions tell users do download DART into a directory called 'DART' in their current directory?"

Previous documentation pages do detail how to download DART into the appropriate directory, but I think I should make a comment about where to find this information if they are missing it and how they need to navigate to the directory they installed DART in before running the first cd command. 

> Line 49: $ csh quickbuild.csh ?-> $ csh quickbuild.sh or are we waiting until this change has been made?

We will be leaving .csh for now and will go back and make this change to all of the documentation when the changes in the code have been made.

File: guide/verifying-installation.rst [hunk 3]
```diff
@@ -27,66 +26,131 @@ Your results should start out looking VERY SIMILAR and may diverge with
...
-Usually, we launch MATLAB from the ``DART/models/lorenz_63/work``
-directory and use the MATLAB *addpath* command to make the
-``DART/matlab/`` functions available for execution in any working
-directory.
+run some of the functions available in ``DART/diagnostics/matlab/``. If
+you have yet to download DART, see :doc:`Downloading DART <downloading-dart>` for
+instructions on how to do so. Usually, we launch MATLAB from the
+``DART/models/lorenz_63/work`` directory. First, open your terminal and navigate
+to the directory where you downloaded DART. Then, execute the following command
+in your terminal to navigate to the``DART/models/lorenz_63/work`` directory.
```
Comment (hkershaw-brown) [inline guide/verifying-installation.rst:34]: rst formatting
```suggestion
in your terminal to navigate to the ``DART/models/lorenz_63/work`` directory.
```

**Review by hkershaw-brown — APPROVED** [review]
Looks great! approved

**Comment by hkershaw-brown** [comment]
@mjs2369 if you have time this afternoon we can merge and release this. (following https://ncar.github.io/dart-developers-guide/releases.html just with a second pair of eyes for your first release)

Otherwise let's merge and release on Monday.

**Review by kdraeder — APPROVED** [review]
I made a couple small changes after the second "trajectory file",
which can be included or ignored.  But they're not showing up in the Conversation, so  I don't know their status.
They just make lines 124-5 more consistent with lines 117-8.

**Pull request merged**